### PR TITLE
Jupiter 10.2.0 cr 198603 group user

### PR DESCRIPTION
**Pull request by assafb**
@rkreich  - please review (doesn't includes entitlement changes yet)

### CODE REVIEW

File: api_v3/lib/types/filters/KalturaGroupUserFilter.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * @package api
+ * @subpackage filters
+ */
+class KalturaGroupUserFilter extends KalturaGroupUserBaseFilter
```
Comment (tan-tan-kanarek) [inline api_v3/lib/types/filters/KalturaGroupUserFilter.php]: @assafb, please validate that either group id or user id are filtered.

File: api_v3/services/GroupUserService.php [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+		//verify group exists
+		$kgroup = kuserPeer::getKuserByPartnerAndUid(  $partnerId, $groupId ,false, KuserType::GROUP);
+		if (! $kgroup)
+			throw new KalturaAPIException ( KalturaErrors::GROUP_NOT_FOUND, $groupId );
+
+		$dbKuserKgroup = KuserKgroupPeer::getByKuserIdAndKgroupId($kuser->getId(), $kgroup->getId());
+		if (!$dbKuserKgroup)
+			throw new KalturaAPIException(KalturaErrors::GROUP_USER_DOES_NOT_EXISTS, $userId, $groupId);
+
+		$dbKuserKgroup->setStatus(KuserKgroupStatus::DELETED);
```
Comment (tan-tan-kanarek) [inline api_v3/services/GroupUserService.php]: @assafb, why won't we really delete it?

File: deployment/base/sql/01.kaltura_ce_tables.sql [hunk 3]
```diff
@@ -2316,4 +2316,28 @@ CREATE TABLE  IF NOT EXISTS `drm_key`
 	`updated_at` DATETIME,
 	PRIMARY KEY (`id`),
 	UNIQUE KEY `partner_id_object_id_object_type_provider` (`partner_id`, `object_id`, `object_type`, `provider`)
-)ENGINE=InnoDB DEFAULT CHARSET=utf8;
\ No newline at end of file
+)ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE `kuser_kgroup`
```
Comment (tan-tan-kanarek) [inline deployment/base/sql/01.kaltura_ce_tables.sql]: @assafb, please also add type to kuser table.

File: api_v3/lib/types/filters/KalturaGroupUserFilter.php (hunk 1 above)
Comment (assafb) [inline api_v3/lib/types/filters/KalturaGroupUserFilter.php]: @tan-tan-kanarek  - I do the validate in groupUserService->list

**Pull request closed without merging**